src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,39 @@
+package Duke.Task;
+
+/**
+ * A deadline is a task that need to be done before a specific date/time.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    /**
+     * The Deadline class constructor has 2 parameters:
+     * the description about a specific deadline and the time that the deadline needs to be done.
+     * @param description The description about the deadline.
```

Comment:
Perhaps there should be an empty line between the description and parameter section? I have noticed the same issue in several other places too